ts/src/binance.ts

```diff
@@ -4200,8 +4200,9 @@ export default class binance extends Exchange {
         const triggerPrice = this.safeValue2 (params, 'triggerPrice', 'stopPrice');
         const stopLossPrice = this.safeValue (params, 'stopLossPrice', triggerPrice);  // fallback to stopLoss
         const takeProfitPrice = this.safeValue (params, 'takeProfitPrice');
-        const isStopLoss = stopLossPrice !== undefined;
-        const isTakeProfit = takeProfitPrice !== undefined;
+        const trailingDelta = this.safeValue (params, 'trailingDelta');
```

Comment:
@blair I'm not sure what this edit intends to fix, can you give an example?

For instance I tested with `binance createOrder "LTC/USDT" STOP_LOSS_LIMIT buy 10 20 '{"trailingDelta":10}'` and seems to be working

Reply:
@blair I understand and agree, I just think we should default to `isStopLoss` only (and not `istakeProfit`) for consistency's sake. I will update that and improve the docs